framework/Samplers/LimitSurfaceSearch.py

```diff
@@ -432,7 +432,7 @@ def localStillReady(self,ready):
     if not ready:
       return ready #if we exceeded the limit just return that we are done
     if type(self.lastOutput) == dict:
-      if not self.limitSurfacePP.ROM.amITrained:
+      if self.lastOutput == None and not self.limitSurfacePP.ROM.amITrained:
```

Comment:
Why you need to check ```self.lastOutput==None``` since another check ```if type(self.lastOutput) == dict:```  is used before?

Reply:
I obviously had the same thought, but this broke the magic for LSS. Removing the None check was causing the MAAP5 AHDET test to fail apparently.

Reply:
@alfoa Can you comment on this?

Reply:
Simply because that if statement always needs to return False (i.e. that early return should not ever happen...). The modification removing the check on self.lastOutput == None, allowed that early return to happen...It was there for the Adaptive DET but I can create another PR to remove it.

Reply:
Thanks, it was very confusing for me when I saw it.